lib/ansible/plugins/filter/log.yml

```diff
@@ -3,19 +3,20 @@ DOCUMENTATION:
   version_added: "1.9"
   short_description: log of (math operation)
   description:
-    - math operation that returns the N logarithim of inputed number logN(X)
+    - Math operation that returns the L(logarithm, https://en.wikipedia.org/wiki/Logarithm) to base N of the input number.
+    - By default, computes the L(natural logarithm, https://en.wikipedia.org/wiki/Natural_logarithm).
   notes:
-    - This is a passthrough to Python's C(math.log)
+    - This is a passthrough to Python's C(math.log).
   positional: _input, base
   options:
     _input:
-      description: number to operate on
-      type: int
+      description: Number to operate on.
+      type: float
```

Comment:
Flagging this to be sure it's correct. This is the only change I've noticed so far to the meaning of the documentation, as opposed to the grammar or syntax.

Reply:
`math.log` works with both floats and integers as inputs, and the output is always a float. So technically neither `int` nor `float` is correct here, but since for most folks ints are pratically a subset of floats, `float` is more correct.